native/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ set(ENGINE_NAME cocos_engine)
 cc_set_if_undefined(CC_DEBUG_FORCE           OFF)
 cc_set_if_undefined(USE_SE_V8                ON)
 cc_set_if_undefined(USE_SE_SM                OFF)
+cc_set_if_undefined(USE_SE_NAPI              OFF)
```

Comment:
So default is use v8 on openarhmony?

Reply:
Default is NAPI, Could refer to https://github.com/cocos/creator-runtime-extensions/pull/470.

cc_set_if_undefined means that `if it's not defined, then use this value, if it's defined, use the defined value`.

And for openharmony, both USE_SE_V8 and USE_SE_NAPI are defined. 
The default is USE_SE_V8=OFF, USE_SE_NAPI=ON

Reply:
Ok